cadence/contracts/test/EVMDeployer.cdc

```diff
@@ -0,0 +1,20 @@
+import "EVM"
+
+/// This contract is intended for testing purposes for the sake of capturing a deployed contract address while native
+/// `evm.TransactionExecuted` event types are not available in Cadence testing framework. The deploying account should
```

Comment:
Soon enough there will be no need for such workarounds :pray: . I already have a WIP PR in https://github.com/onflow/cadence-tools/pull/330